tests/mocks/repository/lib/mock_kbb_vbb.c

```diff
@@ -1,23 +1,31 @@
 /* SPDX-License-Identifier: Apache-2.0 */
```

Comment:
Didn't rename the file, but I feel like I want to pull these out into their own files eventually.